tests/rkt_trust_test.go

```diff
@@ -84,4 +84,16 @@ func TestTrust(t *testing.T) {
 	t.Logf("Now both images can be executed\n")
 	runImage(t, ctx, imageFile, "Hello", false)
 	runImage(t, ctx, imageFile2, "Hello", false)
+	t.Logf("Skip trusted key (rkt trust --skip-trusted) with trusted key absent\n")
+	runRktTrustSkipTrustedTrue(t, ctx, "rkt-prefix.com/my-app", 1, false)
```

Comment:
You pass the parameter `alreadyTrusted==false` but the key is already trusted from the previous tests (line 67 and line 82). You could do your new test on a different image with a different prefix that is not yet trusted (e.g. `imageFile3`, see example line 36). The new test would have to be done before the "rkt trust --root".